### PR TITLE
AAP-20548pt4: Managing automation content updates 

### PR DESCRIPTION
**Pull request by hherbly**
See [AAP-20548](https://issues.redhat.com/browse/AAP-20548) for more info.

### CODE REVIEW

File: downstream/assemblies/hub/assembly-container-user-access.adoc [hunk 1]
```diff
@@ -9,17 +9,17 @@ ifdef::context[:parent-context: {context}]
...
 
 include::hub/ref-container-permissions.adoc[leveloffset=+1]
 include::hub/proc-create-groups.adoc[leveloffset=+1]
-include::hub/proc-assigning-permissions.adoc[leveloffset=+1]
+// [hherbly]: proc-assigning-permissions seems to repeat proc-create-groups.adoc include::hub/proc-assigning-permissions.adoc[leveloffset=+1]
 
 .Additional resources
 
-* See <<container-registry-group-permissions, Container registry group permissions>> to learn more about specific permissions.
+* See <<container-registry-group-permissions, Container registry team permissions>> to learn more about specific permissions.
```
Comment (dcdacosta) [inline downstream/assemblies/hub/assembly-container-user-access.adoc:20]: Is this supposed to be a link?

**Review by dcdacosta — COMMENTED** [review]
One minor comment, otherwise, LGTM!

File: downstream/assemblies/hub/assembly-container-user-access.adoc (hunk 1 above)
Comment (hherbly) [reply downstream/assemblies/hub/assembly-container-user-access.adoc:20]: I'm just going to remove this. It points to the module right before this one, which doesn't give any more "specific" information than what the user has just read. Thanks for catching it!

**Review by dcdacosta — APPROVED** [review]
LGTM!

**Pull request merged**